pact/concrete-policies/collection-policy/collection-policy-v1.pact

```diff
@@ -6,7 +6,7 @@
   @doc "Collection token policy."
 
   (defcap GOVERNANCE ()
-    (enforce-guard (keyset-ref-guard 'marmalade-admin)))
+    (enforce-keyset "marmalade.marmalade-admin"))
```

Comment:
About the namespace, please can you see this:
https://github.com/kadena-io/marmalade/issues/104
It may be related.
I mean that this could be replaced by something like this
```pact
(defconst ADMIN-KEYSET:string (concat [(read-string 'ns), ".", "marmalade-admin"]))
 
 (defcap GOVERNANCE ()
    (enforce-keyset ADMIN-KEYSET))
```



IMHO:
`(enforce-guard "keyset-name")` is sugar syntax allowed by Pact, but should not be used. Because it's confusing: "keyset-name" is not really a guard. Isn't it ?

I would prefer to use:
`(enforce-guard (keyset-ref-guard "keyset-name"))`

or directly (what I did):
`(enforce-keyset "keyset-name")`

We agree that the 3 above syntax are equivalent. What's your opinion ?